lib/config/httpRoutes.js

```diff
@@ -50,7 +51,6 @@ module.exports = [
   {verb: 'post', url: '/admin/_loadMappings', controller: 'admin', action: 'loadMappings'},
   {verb: 'post', url: '/admin/_loadSecurities', controller: 'admin', action: 'loadSecurities'},
 
-  {verb: 'get', url: '/:index/:collection', controller: 'document', action: 'search'},
```

Comment:
This looks like a breaking change to me

Reply:
This route was not documented, is it a breaking change?

Reply:
Technically not but why changing the route?

Reply:
It has a conflict with another route when I hit it (with collection update. le SDK chose the last one)
and imho it's less confusing to have the same url for GET as for POST